TrackerConditions/src/AlignedTrackerMaker.cc

```diff
@@ -80,13 +80,12 @@ namespace mu2e {
 	HepTransform panel_to_plane(dv.x(),dv.y(),dv.z(),0.0,0.0,rz);
 
 	// make an intermediate multiplication
-	HepTransform panel_temp = plane_temp * panel_to_plane * align_panel;
+	HepTransform panel_temp = plane_temp * (panel_to_plane * align_panel);
 
 	for(size_t istr=0; istr< StrawId::_nstraws; istr++) {
-	  // need to const cast so we can update geometry - fixme
-	  Straw& straw = const_cast<Straw &>(panel.getStraw(istr));
```

Comment:
I will approve this now.  In a future iteration I recommend adding the non-const accessor to the Panel class.  As before only the two Maker classes ever see a non-const tracker or a non-const panel.